.github/workflows/deploy.yaml

```diff
@@ -173,3 +173,14 @@ jobs:
         run: |
           rm -rf /tmp/.buildx-cache
           mv /tmp/.buildx-cache-new /tmp/.buildx-cache
+
+      - name: Run Trivy vulnerability scanner
+        uses: aquasecurity/trivy-action@master
```

Comment:
Just to consider: maybe using specific version of the action would make sense?

Reply:
done